js/app.js

```diff
@@ -1 +1,316 @@
-console.log('Hello frontend');
+document.addEventListener('DOMContentLoaded', function() {
+
+// Variables
+
+var boardArray = ["","","","","","","","",""];
+
+var count = 0;
+
+var win = 0;
+
+var spaces = document.querySelectorAll('div');
+
+// Event Listeners
+
+addListeners();
+
+function addListeners() {
+
+spaces[0].addEventListener('click', placeXO_0);
+
+spaces[1].addEventListener('click', placeXO_1);
+
+spaces[2].addEventListener('click', placeXO_2);
+
+spaces[3].addEventListener('click', placeXO_3);
+
+spaces[4].addEventListener('click', placeXO_4);
+
+spaces[5].addEventListener('click', placeXO_5);
+
+spaces[6].addEventListener('click', placeXO_6);
+
+spaces[7].addEventListener('click', placeXO_7);
+
+spaces[8].addEventListener('click', placeXO_8);
+
+}
+
+// X or O on Board
+
```

Comment:
Use a loop here if possible to reduce repeating code, or you could create a function for checking the turn and marking X or O so you don't have to repeat that part